CHANGELOG.md

```diff
@@ -232,6 +308,30 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
     * `seqan3::get<seqan3::field::cigar>(record)` => `record.cigar_sequence()`
     * `seqan3::get<seqan3::field::tags>(record)` => `record.tags()`
 
+Header Changes:
+
+```cpp
+#include <seqan3/io/{alignment_file=> sam_file}/format_bam.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/format_sam.hpp>
+#include <seqan3/io/{alignment_file => sam_file/detail/}format_sam_base.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/header.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input_format_concept.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input_options.hpp>
+#include <seqan3/io/{alignment_file/misc => sam_file/sam_flag}.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output_format_concept.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output_options.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/record.hpp>
```

Comment:
```suggestion
#include <seqan3/io/sam_file/record.hpp> [Added]
```
I think this is also a new file, as I haven't found the old one.

Reply:
The old file does not exist anymore because it was not deprecated just moved ( see https://github.com/seqan/seqan3/commit/df9fd5ff64f59fdb124c4a564a4141d1f9cff22b#diff-e47411e318d39974736c3b9a6d879cc276d5570bafb8e79f5b693ebad0dde23b ).
I'm not sure how to handle this.